app/packs/src/admin/generic/Utils.js

```diff
@@ -36,7 +36,7 @@ const inputEventVal = (event, type) => {
 };
 
 const absOlsTermId = val => (val || '').split('|')[0].trim();
-const absOlsTermLabel = val => val.replace(absOlsTermId(val), '').replace('|', '').trim();
+const absOlsTermLabel = val => val?.replace(absOlsTermId(val), '')?.replace('|', '').trim();
```

Comment:
Instead of the ?. operator you could supply a default value just like in the line above :)